File: MPI/mpicheck/tests/functions/3.0/MPI_Win_shared_query.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    int var_1;
    MPI_Aint *var_2;
    int *var_3;
    void *var_4;
    int ret;
    /* calls */
    ret = MPI_Win_shared_query(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Win_shared_query(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Win_shared_query.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
       INTEGER(KIND=MPI_ADDRESS_KIND) var_4
       INTEGER var_5
        call mpi_win_shared_query(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_win_shared_query(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    